flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/join/JoinITCase.scala

```diff
@@ -770,6 +773,214 @@ class JoinITCase(expectedJoinType: JoinType) extends BatchTestBase {
     )
   }
 
+  @Test
+  def testLeftOuterJoinReorder(): Unit = {
+    // This test is used to test the result after join to multi join and join reorder.
+    tEnv.getConfig.set(
+      OptimizerConfigOptions.TABLE_OPTIMIZER_JOIN_REORDER_ENABLED,
+      Boolean.box(true))
+    // Register table with stats to support join reorder,
+    // join order LJ(LJ(LJ(T5, T3), T2), T1) will reorder to RJ(T1, LJ(T2, LJ(T5, T3)))
+    registerCollection(
```

Comment:
could you use ddl to create test tables ? such as: create table xxx, and Catalog#alterTableStatistics ?

Reply:
I create a JIRA to trace this improvements: https://issues.apache.org/jira/browse/FLINK-28866